samples/bluetooth/throughput/README.rst

```diff
@@ -111,8 +111,8 @@ After programming the sample to both kits, complete following steps to test it:
 
 1. |connect_terminal_both_ANSI|
 #. Reset both kits.
-#. Press **Button 1** on the kit to set the kit into central (tester) role.
-#. Press **Button 2** on the other kit to set the kit into peripheral (peer) mode.
+#. Press **Button 1** or type ``central`` in the terminal on the kit to set the kit into central (tester) role.
```

Comment:
```suggestion
#. Press **Button 1** on the first development kit or type ``central`` in the terminal on the first kit to set it into the central (tester) role.
```

Reply:
Maybe one more change from me. Terminal is not actuality `on` the kit, but it is connected to the kit, so maybe:
```
#. Press **Button 1** on the first development kit or type ``central`` in the terminal
   connected to the first kit to set it into the central (tester) role.
```

Reply:
Sounds great, @doki-nordic !